test/Android/Locals/TestScript.xml

```diff
@@ -265,11 +265,11 @@
     <LocalType>DerivedClass *</LocalType>
     <LocalIsExpandable>True</LocalIsExpandable>
     <Local>escaped</Local>
-    <LocalValue RegEx="0x[a-f0-9]{8} &lt;.L.str1&gt; &quot;Hello\\n\tWorld!\\n&quot;">0xa1b7fc26 &lt;.L.str1&gt; "Hello\n	World!\n"</LocalValue>
+    <LocalValue RegEx="0x[a-f0-9]{8} &lt;.L.str.1&gt; &quot;Hello\\n\\tWorld!\\n&quot;">0xa1b7fc26 &lt;.L.str.1&gt; "Hello\n\tWorld!\n"</LocalValue>
```

Comment:
Could you make it work on both Ndk version 11 and 10, since it's still possible that we can run the test cases on Ndk 10 by passing the right parameter?